urwid_readline/readline_edit.py

```diff
@@ -338,18 +340,24 @@ def insert_new_line(self):
         if self.multiline:
             self.insert_text("\n")
 
-    def enable_autocomplete(self, func, key=None):
+    def enable_autocomplete(self, func, key=None, key_reverse=None):
         self._autocomplete_func = func
         self._autocomplete_key = key if key is not None else "tab"
+        self._autocomplete_key_reverse = key_reverse if key_reverse is not None else "shift tab"
```

Comment:
Too long line, please pass the sources through black

```
python3 -m pip install --user black
python3 -m black -l79 .
```

Reply:
What version of black am I supposed to be using? 
I can't use the 179 flags for some reason
I get `Error: no such option: -1`

Reply:
Any version will be okay but the switch is `-l` not `-1`, as in `--line-length`